src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+
+import classNames from 'classnames';
+
+export interface BadgeProps {
+	text: string;
+	type?: 'default' | 'success' | 'error';
```

Comment:
required: false

Reply:
Hoe bedoel je? Die prop is toch niet required?

Reply:
in het storyboard
maar dat is dus de package
je mag dit negeren